lib/version_utils.pm

```diff
@@ -973,3 +974,12 @@ sub is_community_jeos {
     return (get_var('FLAVOR', '') =~ /JeOS-for-(AArch64|RISCV|RPi)/);
 }
 
+=head2 has_selinux_by_default
```

Comment:
IMO there should be a `has_selinux` that returns `get_var('SELINUX', has_selinux_by_default);`. That way it's possible to use `SELINUX=0` to disable it, which we'll need for apparmor tests.

Could also be done in a followup PR though.

Reply:
i would prefer this in another PR, done by the openQA devs, as they know how it works and what to consider